### PR TITLE
pythonPackages.macpack: init at 1.0.3

### PR DESCRIPTION
**Pull request by maximedenes**
- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/macpack/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0xa1ybrkjkbymjw0wrk9x2w03jck5z3330f5ns1g29hlv5yr7h83";
+  };
+
+  doCheck = false;
+
+  meta = {
+    description = "Makes a macOS binary redistributable by searching the dependency tree and copying/patching non-system libraries.";
```
Comment (dotlambda) [inline pkgs/development/python-modules/macpack/default.nix]: https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes

File: pkgs/development/python-modules/macpack/default.nix [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    license = stdenv.lib.licenses.mit;
+  };
```
Comment (dotlambda) [inline pkgs/development/python-modules/macpack/default.nix]: Homepage?

File: pkgs/development/python-modules/macpack/default.nix [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+  pname = "macpack";
+  version = "1.0.3";
+  disabled = !isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0xa1ybrkjkbymjw0wrk9x2w03jck5z3330f5ns1g29hlv5yr7h83";
+  };
+
+  doCheck = false;
```
Comment (dotlambda) [inline pkgs/development/python-modules/macpack/default.nix]: Why?

Comment (maximedenes) [reply pkgs/development/python-modules/macpack/default.nix]: Sorry, was put there by mistake.

File: pkgs/development/python-modules/macpack/default.nix (hunk 2 above)
Comment (maximedenes) [reply pkgs/development/python-modules/macpack/default.nix]: Added.

File: pkgs/development/python-modules/macpack/default.nix (hunk 1 above)
Comment (maximedenes) [reply pkgs/development/python-modules/macpack/default.nix]: Fixed, thanks!

**Comment by dotlambda** [comment]
@maximedenes Do you want to maintain this package?

@GrahamcOfBorg build python2.pkgs.macpack python3.pkgs.macpack

**Comment by dotlambda** [comment]
> Ran 0 tests in 0.000s

Seems like there are no tests, so please redisable them after all:
```
# no tests implemented
doCheck = false;
```

**Comment by maximedenes** [comment]
This script doesn't seem to fully support a Nix-based setup, please wait before merging, I'll investigate more.

**Comment by dotlambda** [comment]
It seems to me that this isn't a library, i.e. it shouldn't be imported using `import macpack`. Therfore, it should use `buildPythonApplication` and be moved out of `pythonPackages`.

**Comment by maximedenes** [comment]
> It seems to me that this isn't a library, i.e. it shouldn't be imported using import macpack. Therfore, it should use buildPythonApplication and be moved out of pythonPackages.

Yes, that's a good point. Anyway, I realized that the way this utility distinguishes system libraries from user-installed ones seems fundamentally incompatible with a Nix-based layout. So I'm going to close this PR. Sorry for the noise.

**Pull request closed without merging**